lib/workers/repository/onboarding/branch/index.ts

```diff
@@ -36,22 +38,17 @@ export async function checkOnboardingBranch(
   setGitAuthor(config.gitAuthor);
   const onboardingPr = await getOnboardingPr(config);
   if (onboardingPr) {
+    logger.debug('Onboarding PR already exists');
     if (config.onboardingRebaseCheckbox) {
       handleOnboardingManualRebase(onboardingPr);
     }
-    logger.debug('Onboarding PR already exists');
-    const { rawConfigHash } = onboardingPr.bodyStruct ?? {};
-    const commit = await rebaseOnboardingBranch(config, rawConfigHash);
-    if (commit) {
-      logger.info(
-        { branch: config.onboardingBranch, commit, onboarding: true },
-        'Branch updated'
+    if (await scm.isBranchModified(config.onboardingBranch!)) {
+      invalidateExtractCache(config.baseBranch!);
+      isConflicted = await scm.isBranchConflicted(
+        config.baseBranch!,
+        config.onboardingBranch!
       );
     }
-    // istanbul ignore if
-    if (platform.refreshPr) {
-      await platform.refreshPr(onboardingPr.number);
-    }
```

Comment:
I don't recall what this code was for, but why is it removed?

Reply:
Thansks for this . Seem like its needed for a bbs issue : I will add it back.
ref: https://github.com/renovatebot/renovate/issues/3257#issuecomment-466636906
@viceice Can you also confirm?

Reply:
yes, it's need for bitbucket server, so better add a comment and the issue number to the condition

Reply:
The bbs issue won't effect now since we don't push to the onboardingBranch after its created.